tests/DoctrineORMModuleTest/Service/DBALConnectionFactoryTest.php

```diff
@@ -154,4 +155,36 @@ public function testDoctrineAddCustomCommentedType()
         $this->assertInstanceOf(\DoctrineORMModuleTest\Assets\Types\MoneyType::class, $type);
         $this->assertTrue($platform->isCommentedDoctrineType($type));
     }
+
+    public function testGettingPlatformFromContainer()
+    {
+        $config = [
+            'doctrine' => [
+                'connection' => [
+                    'orm_default' => [
+                        'driverClass'   => \Doctrine\DBAL\Driver\PDOSqlite\Driver::class,
```

Comment:
For all instances where you give the fully qualified namespace please replace with a `use` at the top of the script and just the class name in the test.